src/views/Practice/Practice.jsx

```
@@ -22,84 +22,84 @@ import { MessageDialog } from "../../components/Assesment/Assesment";
+import Or from "../../assets/Or.wav";
+import Ange from "../../assets/ange.wav";
+import Ta from "../../assets/Ta.wav";
+import Ble from "../../assets/ble.wav";
+import Le from "../../assets/Le.wav";
+import Mon from "../../assets/mon.wav";
+import Bas from "../../assets/Bas.wav";
+import Ket from "../../assets/ket.wav";
+import Tun from "../../assets/Tun.wav";
+import Nel from "../../assets/nel.wav";
+import Sun from "../../assets/Sun.wav";
+import Set from "../../assets/set.wav";
```

Biome (1.9.4)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)